software/control/microcontroller.py

```diff
@@ -568,20 +577,26 @@ def configure_actuators(self):
         self.wait_till_operation_is_completed()
         self.set_leadscrew_pitch(AXIS.Z,SCREW_PITCH_Z_MM)
         self.wait_till_operation_is_completed()
+        self.set_leadscrew_pitch(AXIS.W,SCREW_PITCH_W_MM)
+        self.wait_till_operation_is_completed()
```

Comment:
@veerwang this should only happen when the microscope has Squid filter wheel

Reply:
@veerwang These should be in a different function

Reply:
Sure